client/my-sites/plans/ecommerce-trial/upgrade-confirmation/confirmation-task/style.scss

```diff
@@ -1,4 +1,32 @@
 .confirmation-task__card {
+	--subtitle-font-size: 0.8125rem;
+	border-radius: 2px;
+	box-shadow: 0 0 0 1px var(--color-neutral-5);
+	cursor: pointer;
+
+	&:hover {
+		box-shadow: 0 0 0 2px var(--color-accent-50);
+	}
+
 	margin: 0;
 	padding: 40px 25px;
```

Comment:
It doesn't look like you've implement the mobile styles, where the padding shifts to `32px` all round.